internal/web/datasethandler.go

```diff
@@ -268,6 +274,13 @@ func (handler *datasetHandler) getEntitiesHandler(c echo.Context) error {
 		return c.NoContent(http.StatusNotFound)
 	}
 
+	// check if we need to return JSON-LD
+	asJsonLd := false
+	acceptHeader := c.Request().Header.Get("Accept")
+	if strings.Contains(acceptHeader, "application/ld+json") {
+		asJsonLd = true
+	}
+
 	preStream := func() error {
 		c.Response().Header().Set(echo.HeaderContentType, echo.MIMEApplicationJSON)
```

Comment:
should we set a different content-type for json-ld?

Reply:
Agreed. Have fixed. We should discuss if we need a refined content type that indicates this is streaming json.